starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/GroceryViewHolder.java

```diff
@@ -0,0 +1,23 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.TextView;
+
+import org.w3c.dom.Text;
+
+/**
+ * Created by NikitaShuvalov on 10/25/16.
+ */
+
+public class GroceryViewHolder extends RecyclerView.ViewHolder {
+    TextView textView0, textView1, textView2, textView3;
```

Comment:
These should have descriptive names like `nameTextView`, `descriptionTextView`, etc.